steps/python/python.docker.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  dockerRegistryConnector: "tomtom-docker-registry-bintray"
+  dockerRepo: "python"
+  dockerDeployFile: "Dockerfile"
+
+steps:
+
+- task: Docker@2
```

Comment:
I don't really have any clue about writing the azure templates. What does `@2` do?

Reply:
This is how Microsoft versions their tasks. It is usually just an incremented number.